js/src/enum.ts

```diff
@@ -137,8 +137,7 @@ export enum MessageHeader {
  * nested type consisting of other data types, or another data type (e.g. a
  * timestamp encoded as an int64).
  *
- * **Note**: Only enum values 0-18 (NONE through Duration) are written to an Arrow
- * IPC payload.
+ * **Note**: Only positive enum values are written to an Arrow IPC payload.
```

Comment:
```suggestion
 * **Note**: Only non-negative enum values are written to an Arrow IPC payload.
```
Maybe this is pedantic; does `0` get written to IPC as the original comment suggested?